src/plugins/auto/plugin.hpp

```diff
@@ -12,6 +12,7 @@
 
 #include <cpp_interfaces/interface/ie_iplugin_internal.hpp>
 #include <cpp_interfaces/interface/ie_internal_plugin_config.hpp>
+#include "../../inference/src/cnn_network_ngraph_impl.hpp"
```

Comment:
@tiger100256-hu any elegant way of handling this include?  looks like this folder path is not in include dirs anyway